Makefile

```diff
@@ -42,6 +42,7 @@ BIN_DIR = $(CURDIR)/build/_output/bin/
 export GOROOT=$(BIN_DIR)/go/
 export GOBIN = $(GOROOT)/bin/
 export PATH := $(GOBIN):$(PATH)
+export PUSH_IMAGES ?= true
```

Comment:
Can we export this only on the bump target? So we make the scope clearer.

Reply:
totally. 
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/9b24ea335fcd0ce675457680017746453f1db07b..27c9d14d406fb6d36c66be74fbe467ecbe6eda8e)